src/DeFuncto.Core/ActivePatternMatching/ActivePattern.cs

```diff
@@ -3,30 +3,96 @@
 
 namespace DeFuncto.ActivePatternMatching;
 
+/// <summary>
+/// Base class for the active pattern matching.
+/// This bit is not, and never will, be documented on the official documentation.
```

Comment:
I would move the part that talks about jumping to f# out of the summary, as a remark maybe.